### ISSUE TITLE
drop probability

### ISSUE THREAD

**Issue by cowarder** [HHTseng/video-classification#31]
Did you use dropout while training? I noticed that all of the three models, you set 'dropout_p=0.0', any reason?

**Comment by LolBeginner** [comment]
Zhaoguo Wang <notifications@github.com>于2020年8月13日 周四18:52写道：

**Comment by a-hassanzadeh** [comment]
I have been playing with this model for a while and training models, initially I had it on 0 and now changed it to default value which is 0.3. My model with the custom data introduced overfits after a number of epcohs so changing it to 0.3 has helped.

**Comment by cowarder** [comment]
I have overfitting too! I played ResRCNN model with my custom dataset(3 number predict, 5000 videos, 1000 classes). if no pretrained parameter, loss doesn't decrease, if initialized with UCF101 parameter, overfitting enve happen from the first epoch(train loss decrease, val loss increase).  Except for dropout prob, did you change any other parameters to relieve overfitting? thanks!

**Comment by a-hassanzadeh** [comment]
5000 videos and 1000 classes means 5 videos per class? I wonder if the model is learning anything?
I am working with 2000 videos and 2 classes and the best I can get is 80% accuracy. I think you might be expecting too much from the model.

**Comment by shumailaahmed** [comment]
> 5000 videos and 1000 classes means 5 videos per class? I wonder if the model is learning anything?
> I am working with 2000 videos and 2 classes and the best I can get is 80% accuracy. I think you might be expecting too much from the model.

i have a similar setting with 1900 videos and 2 classes, i cant get the validation acc to increase beyond 55%, any suggestions for parameters you have tweaked
